modules/nlu/src/backend/api.ts

```diff
@@ -106,6 +107,18 @@ export default async (bp: typeof sdk, nlus: EngineByBot) => {
     }
   })
 
+  router.get('/contexts', async (req, res) => {
+    const botId = req.params.botId
+    const filepaths = await bp.ghost.forBot(botId).directoryListing('/intents', '*.json')
+    const contextsArray = await Promise.map(filepaths, async filepath => {
+      const file = await bp.ghost.forBot(botId).readFileAsObject('/intents', filepath)
+      return file['contexts']
+    })
```

Comment:
Not a big deal, but keeping the api lean would be nice, what do you think of moving the ghost logic to the storage class, like  other ghost calls ?